src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -1354,3 +1365,285 @@ $divider-color: #000; // Color of dividers between assays with targets
+                padding: 5px 30px;
+                min-height: 100px;
+                text-align: center;
+                white-space: nowrap;
+                font-style: italic;
+            }
+        }
+    }
+}
+
+.matrix {
+    &.matrix--rna-seq {
+        padding: 0 20px 20px 0;
+        border-top: 1px solid $matrix-border-color;
+        border-left: 1px solid $matrix-border-color;
+    }
+}
+
+// Inset RNA-seq matrix.
+.matrix__content.matrix__content--encore {
+    .matrix__presentation {
+        display: block;
+
+        @media screen and (min-width: $screen-md-min) {
+            display: flex;
+            align-items: flex-start;
+        }
+
+        // The RNA-seq portion of the page.
+        .matrix__rna-seq {
+            margin-top: 10px;
+            padding: 3px 0 0 3px;
+            background-color: #e8e8e8;
+            border: 1px solid #d0d0d0;
+            margin-bottom: 50px;
+
+            @media screen and (min-width: $screen-md-min) {
+                order: 1;
+                margin-left: 50px;
+                margin-bottom: 0;
+                min-width: 1px; // Prevents flex row overflow by allowing scrolling of this matrix.
+
+                .uaTrident & {
+                    flex-grow: 1;
+                }
+            }
+
+            .matrix__col-category-header {
+                > td {
+                    background-color: transparent;
+                }
+            }
+
+            .category-subheader {
+                @each $term, $header-color, $text-color, $cell-color in $rnaseq-term-name-colors {
+                    &.category-subheader--#{$term} {
+                        background-color: $header-color;
+
+                        a {
+                            color: $text-color;
+                        }
+                    }
+                }
+            }
+
+            .matrix__row-data {
+                > td {
+                    .encore-cell {
+                        @each $term, $header-color, $text-color, $cell-color in $rnaseq-term-name-colors {
+                            &.encore-cell--#{$term} {
+                                background-color: $cell-color;
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        // The ENCORE portion of the page.
+        .matrix__encore {
+            @media screen and (min-width: $screen-md-min) {
+                order: 0;
+
+                .uaTrident & {
+                    flex-grow: 1;
+                }
+            }
+        }
+    }
+}
+
+.type-EncoreMatrix {
+    .matrix-header__target-filter-controls {
+        @media screen and (min-width: $screen-md-min) {
+            flex: 0 1 450px;
+        }
+
+        .general-search-entry {
+            padding-right: 0;
+        }
+    }
+}
+
+.encore-column-spacer {
+    border-left: 1px solid $matrix-border-color;
+    width: $encore-spacer-width;
+}
+
+.type-EncoreMatrix {
```

Comment:
why isn't this part of line 1627?

Reply:
It should have. I missed it as I desperately tried to make rc1.